cdap-ui/app/features/adapters/routes.js

```diff
@@ -64,7 +70,8 @@ angular.module(PKG.name + '.feature.adapters')
             templateUrl: '/assets/features/adapters/templates/tabs/runs.html',
             controller: 'AdapterRunsController',
             ncyBreadcrumb: {
-              label: 'Runs'
+              parent: 'adapters.list',
+              label: '{{$state.params.adapterId | caskCapitalizeFilter}}'
```

Comment:
Ok, I think we removed capitalization from app/dataset/stream/program/etc names from breadcrumb in previous PR. Should we also bring those back?

https://github.com/caskdata/cdap/pull/2318/files#diff-7e907318ef3ddd847ddeea12ade60365R35
https://github.com/caskdata/cdap/pull/2318/files#diff-01a388450052ec0df31f6540b63ee38dR53